library/Exceptions/LengthException.php

```diff
@@ -13,13 +13,21 @@
 
 namespace Respect\Validation\Exceptions;
 
-class LengthException extends ValidationException
+/**
+ * @author Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ * @author Danilo Correa <danilosilva87@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ */
+final class LengthException extends ValidationException
 {
     public const BOTH = 'both';
     public const LOWER = 'lower';
     public const GREATER = 'greater';
     public const EXACT = 'exact';
 
+    /**
+     * {@inheritdoc}
+     */
     public static $defaultTemplates = [
         self::MODE_DEFAULT => [
             self::BOTH => '{{name}} must have a length between {{minValue}} and {{maxValue}}',
```

Comment:
There is also another method in this class (`chooseTemplate()`) that needs the `{@inheritdoc}` annotation.